src/net/sourceforge/kolmafia/request/PlaceRequest.java

```diff
@@ -240,7 +240,9 @@ else if (responseText.contains("a giant jagged hole in the wall")) {
       }
       case "mountains" -> {
         if (responseText.contains("chateau")) {
-          Preferences.setBoolean("chateauAvailable", true);
+          if (!KoLCharacter.inLegacyOfLoathing()) {
+            Preferences.setBoolean("chateauAvailable", true);
```

Comment:
Do you perhaps want to set `replicaChateauAvailable` here?

Reply:
no: 

>If you're in LoL, we don't set Chateau status when viewing mountains, because either:
>
> * you have the real Chateau and may or may not have the replica
> * you've just used the replica
>
>and we can't distinguish the cases.

Reply:
Oh i see, you can see it but can't access in LoL if you have the real one but not the replica? That's a bug

Reply:
Yes. It's the same for NEP too.